src/pages/faq.js

```diff
@@ -19,7 +19,8 @@ function Faq(props) {
   `
 
   return (
-    <GlobalStyles>
+    <div>
+    <GlobalStyles/>
```

Comment:
You only need `GlobalStyles` once in your app. These styles are global → should apply everywhere.
Thus, a perfect place would be `App.js`.

Reply:
will move to PageLayout once my header and footer components are ready. For testing purposes I'll leave it here for now.